src/main/java/io/phasetwo/keycloak/events/WebhookSenderEventListenerProvider.java

```diff
@@ -157,12 +157,12 @@ private ExtendedAdminEvent completeAdminEventAttributes(String uid, Event event)
   }
 
```

Comment:
Hello @FileJunkie . Could you please synch your fork with the p2-inc repository than rebase your branch with main branch? I see that a test `testAdminEventContainsCorrectRealmMasterMaster` is failing

Reply:
Just did that.

Reply:
Thank you @FileJunkie  for this contribution and you patience.